src/IrisConvert/IrisConvert.php

```diff
@@ -163,6 +179,6 @@ public function convertArrayOfSizes()
         }
 
         // add action wp_generate_attachment_metadata
-        add_filter('wp_generate_attachment_metadata', [Iris::class, 'convertImageOnUplodad'], 10, 2);
+        // add_filter('wp_generate_attachment_metadata', [Iris::class, 'convertImageOnUplodad'], 10, 2);
```

Comment:
Should this be disabled still or do we need to ensure it doesn't recall itself within this method?